fluentd/Dockerfile.centos7

```diff
@@ -59,7 +62,9 @@ ADD parser_viaq_docker_audit.rb viaq_docker_audit.rb /etc/fluent/plugin/
 ADD run.sh generate_syslog_config.rb ${HOME}/
 ADD lib/generate_throttle_configs/lib/*.rb ${HOME}/
 ADD lib/filter_parse_json_field/lib/*.rb /etc/fluent/plugin/
-COPY utils/** ${HOME}/utils
+ADD patch/logger/log.rb ${GEM_HOME}/gems/fluentd-0.12.43/lib/fluent
+ADD patch/logger/supervisor.rb ${GEM_HOME}/gems/fluentd-0.12.43/lib/fluent
```

Comment:
maybe add the `/` at the end?  `ADD patch/logger/supervisor.rb ${GEM_HOME}/gems/fluentd-0.12.43/lib/fluent/`
every other place that uses `ADD` for a directory has the trailing `/`